### PR TITLE
[MRG] Shrinkage Transformer

### PR DESCRIPTION
**Pull request by alexandrebarachant**
this transformer can be used to regularize covariance matrices post-estimation.
it's a standard shrinkage, and make use of the `shrunk_covariance` function from sklearn.

@kingjr : any comment ?

### CODE REVIEW

File: pyriemann/estimation.py [hunk 1]
```diff
@@ -419,3 +420,67 @@ def transform(self, X):
...
+    def fit(self, X, y=None):
+        """Fit.
+
+        Do nothing. For compatibility purpose.
+
+        Parameters
+        ----------
+        X : ndarray, shape (n_trials, n_channels, n_samples)
+            ndarray of trials.
+        y : ndarray shape (n_trials, 1)
```
Comment (kingjr) [inline pyriemann/estimation.py]: 1,  not necessary?

File: pyriemann/estimation.py [hunk 2]
```diff
@@ -419,3 +420,67 @@ def transform(self, X):
...
+            labels corresponding to each trial, not used.
```
Comment (kingjr) [inline pyriemann/estimation.py]: Labels

File: pyriemann/estimation.py [hunk 3]
```diff
@@ -419,3 +420,67 @@ def transform(self, X):
...
+
+    def fit(self, X, y=None):
+        """Fit.
+
+        Do nothing. For compatibility purpose.
+
+        Parameters
+        ----------
+        X : ndarray, shape (n_trials, n_channels, n_samples)
+            ndarray of trials.
```
Comment (kingjr) [inline pyriemann/estimation.py]: Target data

File: pyriemann/estimation.py [hunk 4]
```diff
@@ -419,3 +420,67 @@ def transform(self, X):
...
+
+        Returns
+        -------
+        covmats : ndarray, shape (n_trials, n_channels, n_channels)
+            ndarray of covariance matrices for each trials.
+        """
+
+        covmats = numpy.zeros_like(X)
+
+        for i, x in enumerate(X):
```
Comment (kingjr) [inline pyriemann/estimation.py]: ii for explicit disambiguation to complex and avoiding single letter var

**Comment by kingjr** [comment]
Thanks!
